hypothesis-python/docs/community.rst

```diff
@@ -2,9 +2,7 @@
 Community
 =========
 
-The Hypothesis community is small for the moment but is full of excellent people
-who can answer your questions and help you out. Please do join us.
-The major place for community discussion is `the mailing list <https://groups.google.com/forum/#!forum/hypothesis-users>`_.
+The Hypothesis community is full of excellent people who can answer your questions and help you out. Please do join us. The major place for community discussion is `the mailing list <https://groups.google.com/forum/#!forum/hypothesis-users>`_.
```

Comment:
And the [StackOverflow `python-hypothesis`](https://stackoverflow.com/questions/tagged/python-hypothesis)

Reply:
Doh, very next paragraph, grumble :-)

Regardless, I'd argue that mailing lists are a thing of the past for most people in this modern world, and possibly should be relegated to second place.